external/buildscripts/build_all_osx.pl

```diff
@@ -51,13 +51,6 @@
 	'buildusandboo=i'=>\$buildUsAndBoo,
 );
 
-my $monoArch32Target = "i386";
-if ($buildMachine)
-{
-	print(">>> Building $monoArch32Target\n");
-	system("perl", "$buildscriptsdir/build.pl", "--arch32=1", "--clean=1", "--classlibtests=0", @passAlongArgs) eq 0 or die ("failing building $monoArch32Target");
```

Comment:
@joncham don't we still use the 32bit runtime for il2cpp perf or size comparison?

Reply:
I really hope we don't since it doesn't work on latest macOS :P

Reply:
As I understand it the build agents aren't running catalina yet so they can still cross compile 32bit. I poked @joncham about this and we think the path forward is to just disable the il2cpp tests that need the 32bit runtime for now.

Reply:
cc @joshpeterson @brian-taylor-unity @mrvoorhe  we likely need to disable 32-bit tests running on OSX (Android?) until we can mix reference/target arch in tests like pinvoke.

Reply:
Oh yes, that is why we kept the 32-bit slice around. I had forgotten. On CI we run Android tests on Windows now, so I don't think we need this.

We should modify the IL2CPP test framework to give a nice error when Android armv7 tests run on macOS though.